challenges.yml

```diff
@@ -0,0 +1,184 @@
+addsimplenamespacecoding:
```

Comment:
@weaverryan  OK! BTW, `Shift` + `cmd` + `U` shortcut in PhpStorm convert `ClassName` to the `classname`! :)